pycytominer/cyto_utils/cells.py

```diff
@@ -267,14 +292,20 @@ def load_compartment(self, compartment):
         return df
 
     def aggregate_compartment(
-        self, compartment, compute_subsample=False, aggregate_args=None
+        self,
+        compartment,
+        compute_subsample=False,
+        compute_counts=False,
+        aggregate_args=None,
     ):
         """Aggregate morphological profiles. Uses pycytominer.aggregate()
 
         :param compartment: string indicating the specific compartment, defaults to "cells"
         :type compartment: str
         :param compute_subsample: determine if subsample should be computed, defaults to False
         :type compute_subsample: bool
+        :param compute_counts: determine if the number of the objects and fields should be computed, defaults to False
```

Comment:
can you add to this docstring slightly? can you add one example of what an object and field is? I found myself getting lost a couple of lines down as to what "counts" i was computing and I jumped back here. An example would be crystal clear

Reply:
Done - https://github.com/cytomining/pycytominer/pull/129/commits/b81169d3c93ddb9768e16ed14e73fc475d864041